scripts/curate_basel_additional_data_20221028.py

```diff
@@ -0,0 +1,151 @@
+        for file in fileList:
+            if file.endswith('.nii.gz'):
+                originalFilePath = os.path.join(dirName, file)
+                jsonSidecarPath = os.path.join(dirName, file.split(sep='.')[0] + '.json')
+                if not os.path.exists(jsonSidecarPath):
+                    print("Missing: " + jsonSidecarPath)
+                    if file.endswith('lesion-manual.nii.gz'):
+                        data_json_label = {}
+                        data_json_label[u'Author'] = "Katrin"
+                        data_json_label[u'Label'] = "lesion-manual"
+                        with open(jsonSidecarPath, 'w') as outfile:
+                            outfile.write(json.dumps(data_json_label, indent=2, sort_keys=True))
+                        outfile.close()
+                    elif file.endswith("seg-manual.nii.gz"):
+                        data_json_label = {}
+                        data_json_label[u'Author'] = "Katrin"
+                        data_json_label[u'Label'] = "seg-manual"
+                        with open(jsonSidecarPath, 'w') as outfile:
+                            outfile.write(json.dumps(data_json_label, indent=2, sort_keys=True))
+                        outfile.close()
+                    else:
+                        os.system('touch ' + jsonSidecarPath)
+
+    sub_list = os.listdir(path_output)
+    sub_list.remove('derivatives')
+
+    sub_list.sort()
+
+    import csv
+
+    participants = []
+    for subject in sub_list:
+        row_sub = []
+        row_sub.append(subject)
+        row_sub.append('n/a')
+        row_sub.append('n/a')
+        participants.append(row_sub)
+
+    print(participants)
+    with open(path_output + '/participants.tsv', 'w') as tsv_file:
+        tsv_writer = csv.writer(tsv_file, delimiter='\t', lineterminator='\n')
+        tsv_writer.writerow(["participant_id", "sex", "age"])
+        for item in participants:
+            tsv_writer.writerow(item)
+
+    # Create participants.json
+    data_json = {"participant_id": {
+        "Description": "Unique Participant ID",
+        "LongName": "Participant ID"
+        },
+        "sex": {
+            "Description": "M or F",
+            "LongName": "Participant sex"
+        },
+        "age": {
+            "Description": "yy",
+            "LongName": "Participant age"}
+    }
+
+    with open(path_output + '/participants.json', 'w') as json_file:
+        json.dump(data_json, json_file, indent=4)
+
+    # Create dataset_description.json
+    dataset_description = {"BIDSVersion": "BIDS 1.6.0",
+                           "Name": "BIDSify INsIDER_SCT_Segmentations_COR"
+                           }
+
+    with open(path_output + '/dataset_description.json', 'w') as json_file:
+        json.dump(dataset_description, json_file, indent=4)
+
+    # Create README
+    with open(path_output + '/README', 'w') as readme_file:
+        readme_file.write('BIDSify MP2RAGE MS SEG dataset: INsIDER_SCT_Segmentations_COR.')
```

Comment:
there will be a conflict between the existing README and this new README. If you look at the existing README, it has a bunch of useful information, which would be overwritten by this script:
~~~
Dataset for basel-mp2rage.

Dataset shared by: Charidimos Tsagkas <charidimos.tsagkas@usb.ch>
Other collaborators: Christina Granziera <cristina.granziera@usb.ch>
Email communication: 2021-06-07
Original package file: INsIDER_20210607_30patients
Content: MP2RAGE images of the brain and spinal cord with manual segmentation of MS lesions from one expert rater.
~~~

So it might be wise to append to the existing README, eg:

~~~
2022-10-30: Added new data from INsIDER_SCT_Segmentations_COR
~~~

Reply:
It might be good to also point to this repository in the README: https://github.com/ivadomed/model_seg_ms_mp2rage